src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -506,6 +506,7 @@ internal void SelectAll(object parameter)
         {
             DynamoSelection.Instance.ClearSelection();
             Nodes.ToList().ForEach((ele) => DynamoSelection.Instance.Selection.Add(ele.NodeModel));
+            Notes.ToList().ForEach((ele) => DynamoSelection.Instance.Selection.Add(ele.Model));
```

Comment:
I think add range does not work correctly with dynamo Selection for some reason.